integration_test/README.md

```diff
@@ -120,8 +120,8 @@ the test will:
 1.  Wait for up to 7 minutes for logs matching the expectations in
     `applications/<application>/expected_logs.yaml` to appear in the Google
     Cloud Logging backend.
-1.  Wait up to 7 minutes for the metric from
-    `applications/<application>/metric_name.txt` to appear in the Google Cloud
+1.  Wait up to 7 minutes for metrics matching the expectations in
+   `applications/<application>/expected_metrics.yaml` to appear in the Google Cloud
```

Comment:
I believe this needs to line up with `Wait`…

Reply:
Fixed.